validations.rb

```diff
@@ -0,0 +1,31 @@
+module Validations
+    def Validations.validate_amount(amount)
+        if amount > 0
+            return true
+        else
+            return false
+        end
+    end
+
+    def Validations.validate_withdrawal(amount, balance)
+        if amount > balance
```

Comment:
Same here and I noticed similar below (line 25). `amount > balance` is already a boolean, true or false, so you can return the value of `amount > balance`. It's obsolete to do:
```ruby
if boolean
  return true
else
  return false
end
```

when you can do `return boolean`